File: Java/src/IntegerToRomanTest.java
import org.junit.Assert;
import org.junit.Test;

public class IntegerToRomanTest {

    @Test
    public void testIntToRoman() {
        IntegerToRoman i = new IntegerToRoman();

        Assert.assertEquals(i.intToRoman(1), "I");

        Assert.assertEquals(i.intToRoman(621), "DCXXI");

        Assert.assertEquals(i.intToRoman(1996), "MCMXCVI");

        Assert.assertEquals(i.intToRoman(999), "CMXCIX");
    }

}

File: Kotlin/src/IntegerToRomanTest.kt
import org.junit.Test

class IntegerToRomanTest {

    @Test
    fun testIntToRoman() {
        val i = IntegerToRoman()

        assert(i.intToRoman(1) == "I")

        assert(i.intToRoman(621) == "DCXXI")

        assert(i.intToRoman(1996) == "MCMXCVI")

        assert(i.intToRoman(999) == "CMXCIX")
    }

}